mmcv/ops/csrc/parrots/iou3d.cpp

```diff
@@ -134,7 +134,7 @@ void iou3d_nms_forward(Tensor boxes, Tensor keep, Tensor keep_num,
     int64_t *keep_data = keep.data_ptr<int64_t>();
     int64_t *keep_num_data = keep_num.data_ptr<int64_t>();
 
-    const int col_blocks = DIVUP(boxes_num, THREADS_PER_BLOCK_NMS);
+    const int col_blocks = GET_BLOCKS(boxes_num, THREADS_PER_BLOCK_NMS);
```

Comment:
what if actual `col_blocks` is greater than 4096?

Reply:
I think it may be unsafe and we should only use `GET_BLOCKS` for block allocation. I will fix it.

Reply:
Fixed